content/post/Android_Root101.en.md

```diff
@@ -1,31 +1,32 @@
 +++
 title = "A Guide to Flashing Custom ROMs [Android]"
-slug = "Flashing Custom ROMs 101"
+slug = "custom-rom-101"
 date = "2018-02-25T00:00:00"
+image = "img/rohanvtk/csrom.jpg"
 description = "Sick and tired of the bloatware on your current OEM ROM. Head over and learn how to flash custom ROMs."
 author = "RohanVTK"
-tags = ['Smartphone', 'Customization']
+tags = ['Smartphone', 'Customization','Android']
 categories = ['Android']
 +++
 
-**_(This blog starts off with a warning: Although the process is super easy and there hardly is a chance that you will brick your device during the process but, it's my responsibility to make you aware of it)_**
+{{<redwarntxt>}}This blog starts off with a warning: Although the process is super easy and the chances of your phone getting bricked(useless) are negligible but, it's my responsibility to make you aware of it{{</redwarntxt>}}
```

Comment:
For stuff like this probably you should also consider adding images. That show how it looks now.

Reply:
I want to add images. Is there a way to resize images in hugo?
The current images are going way off the limits. 😭

Reply:
Same as we add images in markdown as Hugo uses Blackfriday Markdown parser 
`![](IMAGE_URL)`

For resizing images, you can try [these hacks](https://gist.github.com/uupaa/f77d2bcf4dc7a294d109) if you are using Markdown. (Do test them on your local deployment first)
Or just sprinkle some HTML.

![](https://cultofthepartyparrot.com/parrots/hd/gentlemanparrot.gif)

Reply:
Party parrot complimentary with every PR 

:hatching_chick: <<<<< Party Parrot

Reply:
I created a shortcode file for that. CSS helped 😄 But, not center aligned. 😢

Reply:
@Sc0rpi0n101 Didn't you have all images as center aligned.

Reply:
not all, I provided an option to turn images center-aligned using `#center`

Reply:
@Sc0rpi0n101 using the shorthand method?

Reply:
`[](../abc.jpg#center)`